app/renderer/src/assets/locales/en/translation.json

```diff
@@ -282,5 +282,14 @@
   "invalidCapType": "Invalid capability type: {{type}}",
   "whitespaceDetected": "Text Starts and/or Ends With Whitespace",
   "pcloudyCredentialsRequired": "pCloudy username and api key are required!",
-  "duplicateCapabilityNameError": "A capability set with the same name already exists"
+  "duplicateCapabilityNameError": "A capability set with the same name already exists",
+  "errorLoadingGestures": "Error loading gestures",
+  "unableToUploadGestureFiles": "Could not upload gesture from below files:",
+  "gestureEmptyTickError": "Actions should have valid ticks.",
+  "gestureInvalidEventError": "Invalid event type {{invalidEvent}}. Valid values are {{validEvents}}.",
+  "gestureRequiredFieldsError": "Value {{fields}} are required for {{eventType}} event.",
+  "gestureRequiredFieldError": "Value {{field}} is required for {{eventType}} event.",
```

Comment:
Is it worth defining two nearly identical lines? I would just use the plural line for all cases.